content/en/docs/concepts/workloads/pods/pod-quality.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Pod Quality of Service Classes
+content_type: concept
+weight: 30
+---
+
+<!-- overview -->
+
+This page explains how Kubernetes assigns Quality of Service (QoS) classes to
+Pods as a side effect of specifying resource constraints on Containers. QoS
+classes are by Kubernetes to make decisions about which Pods to evict when
+there are not enough resources on a Node.
+
+<!-- body -->
+
+## Quality of Service Classes
+
+Because Kubernetes co-locates all Containers within a given {{< glossary_tooltip text="Pod" term_id="pod" >}}
+to the same {{< glossary_tooltip text="Node" term_id="node" >}}, it uses a Pod-level
+classification to determine how to meet
+[resource requirements](docs/concepts/configuration/manage-resources-containers/).
+This is known as {{< glossary_tooltip text="Quality of Service" term_id="qos-class" >}}
+(QoS) class. Kubernetes assigns every Pod a QoS class based on the resource requests
+and limits of its component {{< glossary_tooltip text="Containers" term_id="container" >}}.
+QoS classes are used by Kubernetes to decide which Pods to evict from a Node experiencing
+[Node Pressure](/docs/concepts/scheduling-eviction/node-pressure-eviction). The possible
+QoS classes are Guaranteed, Burstable, and BestEffort. When a Node runs out of resources,
+Kubernetes will first evict BestEffort Pods running on that Node followed by Burstable and
+Guaranteed Pods. When this eviction is due to resource pressure, only Pods exceeding their
+resource requests are candidates for eviction.
+
+### Guaranteed
+
+Guaranteed Pods have the strictest resource limits and the highest priority to avoid
+being evicted. They are guaranteed not to be killed until they exceed their limits
+or there are no lower-priotiy Pods that can be evicted from the Node. They may
+not acquire resources beyond their specified limits. These Nodes can also make
+use of exclusive CPUs using the
+[static](/docs/tasks/administer-cluster/cpu-management-policies/#static-policy) CPU management policy.
+
+#### Criteria
+
+For a Pod to be given a QoS class of Guaranteed:
+
+* Every Container in the Pod must have a memory limit and a memory request.
+* For every Container in the Pod, the memory limit must equal the memory request.
+* Every Container in the Pod must have a CPU limit and a CPU request.
+* For every Container in the Pod, the CPU limit must equal the CPU request.
+
+### Burstable
+
+Burstable Pods have some lower-bound resource guarantees based on the request, but
+do not require a specific limit. If a limit is not specified they default to a
+limit equivalent to the capacity of the Node, which allows them to flexibly increase
+their resources if resources are available. In the event of Pod eviction due to Node
+resource pressure, these Pods are evicted only after all Best-Effort Pods are evicted.
+
```

Comment:
```suggestion
Because a `Burstable` Pod can include a container that has no resource limits or requests, a Pod
that is `Burstable` can try to use any amount of node resources.

```